content/docs/visualization/template_reference.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Template reference
+sort_rank: 6
+---
+
+# Template reference
+
+Prometheus has support for templating in both the summary/description field of
+alerts and also exposed as html endpoints. Templates have the ability to run
+queries against the databases, iterate over data, use conditionals, format data
+etc. This is based on the [go templating](http://golang.org/pkg/text/template/)
+system.
+
+## Data Structures
+
+The primary structure is the sample, defined as:
```

Comment:
On Tue, Jan 13, 2015 at 12:45 PM, brian-brazil <notifications@github.com>
wrote:

> In content/docs/visualization/template_reference.md
> <https://github.com/prometheus/docs/pull/5#discussion-diff-22857092>:
>
> > +---
> > +title: Template reference
> > +sort_rank: 6
> > +---
> > +
> > +# Template reference
> > +
> > +Prometheus has support for templating in both the summary/description field of
> > +alerts and also exposed as html endpoints. Templates have the ability to run
> > +queries against the databases, iterate over data, use conditionals, format data
> > +etc. This is based on the [go templating](http://golang.org/pkg/text/template/)
> > +system.
> > +
> > +## Data Structures
> > +
> > +The primary structure is the sample, defined as:
>
> I meant that what you propose reads as docs for the query function, what
> I currently have isn't specific to any given function.
>
Ok, but there needs to be some more context than just "The primary
structure is the sample". Maybe you can describe to me more elaborately in
which contexts this type is relevant and I can suggest a better wording :)